Docs/ClippingPrimitives.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Clipping Primitives
```

Comment:
>Clipping Primitives [](start = 7, length = 19)

The whole document is super-easy to follow and very informative

Reply:
Thank you for the kind words!